### ISSUE TITLE
Would you like to transfer the list to RegExps?

### ISSUE THREAD

**Issue by tunnckoCore** [aloisdg/awesome-regex#11]
Can send invite. Then in one future point we can generate our main wiki page from that list - im thinking for some github webhook; when PR is accepted to update the wiki. Same way can be for future website, which will include more things - like this list, about the project, list of repos / members and etc.

**Comment by huei90** [comment]
It's @aloisdg's call.

**Comment by aloisdg** [comment]
@tunnckoCore It is a nice idea. If the project keep growing, I am fine. I think this transfer will help. I am still in vacation, can we talk again about this later. I dont think we are hurry :)

**Comment by huei90** [comment]
I agree with @aloisdg. `awesome-regex` is still a baby yet, we are waiting for it to become a **real** man.
We will see the transfer in the future when everything is ready.

**Comment by tunnckoCore** [comment]
> I dont think we are hurry :)

yea, no problem. :) im busy to the end of month.
